cylc/flow/config.py

```diff
@@ -180,14 +181,11 @@ def __init__(
         self.initial_point: Optional['PointBase'] = None
         self.start_point: Optional['PointBase'] = None
         self.final_point: Optional['PointBase'] = None
-        self.first_graph = True
         self.clock_offsets = {}
         self.expiration_offsets = {}
         self.ext_triggers = {}  # Old external triggers (client/server)
         self.xtrigger_mgr = xtrigger_mgr
         self.workflow_polling_tasks = {}  # type: ignore # TODO figure out type
-        self._last_graph_raw_id: Optional[tuple] = None
-        self._last_graph_raw_edges = []  # type: ignore # TODO figure out type
```

Comment:
This takes out the old graph cashing system, I guess we don't need it anymore?

(could quickly reimplement via `lru_cache` if ever needed again [via a partial or something like that to ignore the non-serialisable `self` argument]).

Reply:
I meant to comment on this, sorry. I was thinking we'd need something different for the Cylc 8 graph view, with delta updates via the UIS.  And even if we have to regenerate all graph edges from scratch on each change, it'll be push (to a single UIS) so we still won't need caching in case of another request for the full graph.  Haven't properly thought it through though (damn, that's a lot of *ough* in a row). Easy enough to restore those lines for now, just in case. What do you think?

Reply:
Yeah, not too sure how we are going to unroll the graph in the n>1 window, ideally via some incremental approach at the UIS (would require the UIS to load the config)? Nothing much is using `get_graph_raw` at the moment so I think we can afford to loose cashing.

Easy to reimplement using standard tooling:

```python
@staticmethod
@lru_cache(cache_size=1)
def _get_graph_raw(_, *args):
    ...

def get_graph_raw(self, *args):
    return self._get_graph_raw(id(self), *args)
```